conans/client/configure_environment.py

```diff
@@ -98,8 +98,8 @@ def _gcc_env(self):
 
     @property
     def command_line_env(self):
-        if self.os == "Linux" or self.os == "Macos" or self.os == "FreeBSD":
-            if self.compiler == "gcc" or "clang" in str(self.compiler):
+        if self.os == "Linux" or self.os == "Macos" or self.os == "FreeBSD" or self.os == "SunOS":
+            if self.compiler == "gcc" or "clang" in str(self.compiler) or "cc" in str(self.compiler):
```

Comment:
sure, we can change it, but do we map the Solaris Compiler ("cc") to that name then?